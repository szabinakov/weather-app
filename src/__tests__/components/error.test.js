import React from 'react'
import { render } from '@testing-library/react'
import ErrorMessage from '../../components/error'

describe('The errormessage', () => {
    it('matches the screenshot', () => {
        const { asFragment } = render (
        <ErrorMessage 
            errorMessage= 'This is the error message'
        />)

        expect(asFragment).toMatchSnapshot()
     })
    
    it('renders the error message', () => {
        const { getByTestId } = render (<ErrorMessage/>)
        expect(getByTestId('errorId')).toHaveClass('error-message')
    })
})