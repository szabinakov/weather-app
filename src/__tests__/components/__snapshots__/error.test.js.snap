// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The errormessage matches the screenshot 1`] = `[Function]`;
